src/docs/truffle/reference/configuration.md

```diff
@@ -51,6 +51,135 @@ Any of the following solutions will remedy this issue:
 
 ## General options
 
+### SolidityLog
+
+Like javascript's console.log, you can utilize the same concept to print state
+of your contract during execution. This log will appear when you use `truffle
+test` or `truffle develop`. To use it you would import `truffle/console.sol`
+into your solidity contract and use it like you would in JavaScript. See the 
+[Ganache documentation](https://github.com/trufflesuite/ganache/tree/develop/src/chains/ethereum/console.log#ganacheconsolelog) for details.
+
+<details><summary>Consider this solidity contract</summary>
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.25 <0.9.0;
+
+import "truffle/console.sol";
+
+contract Telephone {
+    uint public secret;
+
+    constructor(uint _secret) payable { 
+        console.log("CSTR::Telephone\n\tsecret: %o", _secret); 
+        secret = _secret;
+    }
+
+    function derivedSecret() public view returns (uint256) {
+        uint256 telephoneSecret = 100 * secret + 1;
+        console.log("The passphrase is: %o", secret);
+        console.log("Leak it a hundred times, plus one: %o", telephoneSecret);
+        return telephoneSecret; 
+    }
+}
+```
+</details>
+
+<details><summary>and this JavaScript test file</summary>
+
+```javascript
+const Telephone = artifacts.require("Telephone");
+
+const getRnd = function() {
+  const [min, max] = [13, 1013];
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+contract("Telephone", function(/* accounts */) {
+  let subj;
+  describe("Works with new deployment", function() {
+    let newSecret;
+    before(async function() {
+      newSecret = getRnd();
+      subj = await Telephone.new(newSecret);
+    });
+    
+    it("Should have a deployed contract", async function() {
+      return assert.isTrue(subj !== undefined);
+    });
+    
+    it("has the secret", async function() {
+      const best = await subj.secret();
+      return assert.equal(newSecret, best);
```

Comment:
Should this be

```suggestion
      assert.strictEqual(best, newSecret);
```

? Since the actual value is _best_, the expected value is _newSecret_? Also, `strictEqual` is probably the more idiomatic function to use here.